ReSharper.FSharp/test/data/cache/csharpResolve/Properties 22 - Accessor modifiers - Private getter - Auto.gold

```diff
@@ -4,12 +4,11 @@
     {
         var t = new T();
         var _ = t.|A|(0);
-        t.|A|(1) = 1;
+        t.A = 1;
     }
 }
 ---------------------------------------------------------
-(0): ReSharper Underlined Error Highlighting: Cannot access private property 'A' here
-(1): ReSharper Underlined Error Highlighting: Cannot access private property 'A' here
+(0): ReSharper Underlined Error Highlighting: The property 'T.A' cannot be used in this context because the 'get' accessor is inaccessible
```

Comment:
Could you please also add tests with a public getter and a private setter? It seems to be a more popular situation, so it would be great to have a test that dumps the error.